[2024-06-03T04:08:26.767+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T04:08:26.792+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T04:08:02.193035+00:00 [queued]>
[2024-06-03T04:08:26.801+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load manual__2024-06-03T04:08:02.193035+00:00 [queued]>
[2024-06-03T04:08:26.802+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-03T04:08:26.815+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-03 04:08:02.193035+00:00
[2024-06-03T04:08:26.839+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2172) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T04:08:26.841+0000] {standard_task_runner.py:63} INFO - Started process 2174 to run task
[2024-06-03T04:08:26.840+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load', 'manual__2024-06-03T04:08:02.193035+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp9js8u708']
[2024-06-03T04:08:26.842+0000] {standard_task_runner.py:91} INFO - Job 262: Subtask load
[2024-06-03T04:08:26.867+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-03T04:08:26.908+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.load manual__2024-06-03T04:08:02.193035+00:00 [running]> on host aa989158b36e
[2024-06-03T04:08:27.021+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T04:08:02.193035+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T04:08:02.193035+00:00'
[2024-06-03T04:08:27.023+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T04:08:27.056+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/google/cloud/firestore_v1/base_collection.py:302 UserWarning: Detected filter using positional arguments. Prefer using the 'filter' keyword argument instead.
[2024-06-03T04:08:33.416+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T04:08:33.419+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T04:08:33.430+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load, run_id=manual__2024-06-03T04:08:02.193035+00:00, execution_date=20240603T040802, start_date=20240603T040826, end_date=20240603T040833
[2024-06-03T04:08:33.470+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T04:08:33.485+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T04:08:33.488+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
